FlowCryptUI/Resources/Helpers.swift

```diff
@@ -18,3 +20,15 @@ public func testAttributedText() -> NSAttributedString {
         ]
     )
 }
+
+public func getAvatarImage(text: String) -> ASImageNode {
+    let node = ASImageNode()
+    let avatarImage = LetterAvatarMaker()
+        .setCircle(true)
+        .setUsername(text.capitalized)
+        .build()
+    node.image = avatarImage
+    node.style.preferredSize.width = 50
+    node.style.preferredSize.height = 50
```

Comment:
What about making it a bit smaller, for example 44x44, looks better for me:

current:
<img width="200" alt="Screenshot 2023-08-15 at 13 46 22" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/002cbc35-9e4f-410e-a15b-f49a9115b4b1">

44x44:
<img width="200" alt="Screenshot 2023-08-15 at 13 46 58" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/fc96d646-548d-4a36-a91f-60af5f7feb4a">

Reply:
Sounds good!